src/writer/proposal.ts

```diff
@@ -36,6 +37,35 @@ export const getProposalsCount = async (space, author) => {
   return await db.queryAsync(query, [space, author]);
 };
 
+async function validateSpace(space: any) {
+  if (!space) {
+    return Promise.reject('unknown space');
+  }
+
+  if (space.hibernated) {
+    return Promise.reject('space hibernated');
+  }
+
+  if (space?.deleted) return Promise.reject('space deleted, contact admin');
+
+  const schemaIsValid: any = snapshot.utils.validateSchema(snapshot.schemas.space, space, {
+    snapshotEnv: SNAPSHOT_ENV
+  });
+
+  if (schemaIsValid !== true) {
+    const firstErrorObject: any = Object.values(schemaIsValid)[0];
+    if (firstErrorObject.message === 'network not allowed') {
```

Comment:
If this this not true it should still `reject`, all space settings need to be valid

Reply:
All remaining errors from this validation are schema error (missing fields, wrong field type), which should never happen, since the space was fetched from the db directly via `getSpace()`.

All other settings (strategies, validation, etc...) are checked inside `validateSpaceSettings()`

Reply:
I think better to handle everything, instead of adding multiple if conditions in future (we may not remember that this if condition exist) ust in case if we add some other thing similar to networks.